lib/project-data.ts

```diff
@@ -30,7 +30,16 @@ export class ProjectData implements IProjectData {
 	public projectDir: string;
 	public platformsDir: string;
 	public projectFilePath: string;
-	public projectId: string;
+	public projectIdentifiers: IProjectIdentifier;
+	get projectId(): string {
+		this.warnProjectId();
+		return this.projectIdentifiers.ios;
+	}
+	set projectId(identifier: string) {
```

Comment:
is setting project id used only in `initializeProjectData`?

Reply:
Its not used but its intended for hooks and extensions that might try to modify it.

Reply:
please add a comment as this is not easily deducted